docs/decisions/0017-reimplement-asset-processing.rst

```diff
@@ -217,61 +216,50 @@ The three top-level edx-platform asset processing actions are *build*, *collect*
 
        Paver task that invokes ``webpack --watch`` for Webpack assets and watchdog (a Python library) for other assets.
 
-     - ``scripts/build-assets.sh --watch <stage>``
-
-       (where ``<stage>`` is optionally one of the build stages described above. If provided, only that stage's assets will be watched.)
+     - ``npm run watch``
 
        Bash wrappers around invocation(s) of `watchman <https://facebook.github.io/watchman/>`_, a popular file-watching library maintained by Meta. Watchman is already installed into edx-platform (and other services) via the pywatchman pip wrapper package.
 
-       Note: This adds a Python dependency to build-assets.sh. However, we could be clear that watchman is an *optional* dependency of build-assets.sh which enables the optional ``--watch`` feature. This would keep the *build* action Python-free. Alternatively, watchman is also available Python-free via apt and homebrew.
```

Comment:
This note seemed irrelevant now that `npm run watch` is an entirely separate command.